_projects/2025/radis/Exomol_electronic_spectra.md

```diff
@@ -0,0 +1,75 @@
+---
+name: Electronic spectra for RADIS
+desc: The project aims at adding the possibility to calculated electronic spectra with RADIS, mainly using the newly available molecules in ExoMol
+
+# add a short one line description of your project
+requirements:
+# Student requirements:
+ - Knowledge of Pandas or other DataFrame libraries
+difficulty: medium
+issues:
+- https://github.com/radis/radis/issues/724
+mentors:
+# First person in contact; mentors may change before project starts.
+# GitHub handles
+ - minouHub
+ - erwanp
+initiatives:
+ - GSOC
+project_size:
+ - 350 h
+tags:
+# Different technologies needed
+ - Python
+ - Vaex
+ - Pandas
+ - Git
+collaborating_projects:
+# suborganisation(s) to which this project belongs.
+ - radis
+---
+
+
+#### Description
+
+  
+The RADIS code was developed for the characterization of plasmas, flames and atmospheres. High-temperature spectral calculations require to resolve the shape of tens of millions of lines, which is the usual performance bottleneck. RADIS implements a new algorithm to compute these lineshapes, and is already one of the fastest line-by-line spectral codes available. It can also compute many different types of spectra (absorption / emission / equilibrium / nonequilibrium). In a typical calculation, a database of coefficients is loaded and these coefficients are multiplied according to physics laws to generate a set of linestrength in a spectrum. RADIS can handle different molecular databases such as HITRAN, HITEMP, EXOMOL, and GEISA. Currently, it supports the computation of rovibrational spectra, which correspond to transitions between rotational and vibrational energy levels. However, it lacks the capability to compute electronic spectra from databases like EXOMOL, which would complement its existing capabilities.
```

Comment:
```suggestion
The RADIS code was developed for the characterization of plasmas, flames and atmospheres.
High-temperature spectral calculations require to resolve the shape of tens of millions of lines, which is the usual performance bottleneck.
RADIS implements a new algorithm to compute these lineshapes, and is already one of the fastest line-by-line spectral codes available.
It can also compute many different types of spectra (absorption / emission / equilibrium / nonequilibrium).
In a typical calculation, a database of coefficients is loaded and these coefficients are multiplied according to physics laws to generate a set of line strengths in a spectrum.
RADIS can also handle different molecular databases such as HITRAN, HITEMP, EXOMOL, and GEISA.
```